### ISSUE TITLE
Output directory collides with other plugins

### ISSUE THREAD

**Issue by benmccann** [kxbmap/sbt-jooq#25]
A lot of sbt plugins write to `src_managed`. Some delete the whole directory whenever they generate new sources. This makes is difficult to use sbt-jooq with other plugins that generate sources

The preferred way now is to write to `crossTarget.value / "src_managed_jooq"` and add the directory to `managedSourceDirectories in Compile`. This is the recommendation from the sbt team https://github.com/sbt/sbt/issues/1664#issuecomment-213057686. I tried it with another plugin and it seems to work well.

**Comment by kxbmap** [comment]
I think this should be addressed by a misbehaving plugin, not sbt-jooq.
But I'll consider the workaround presented.

**Comment by kxbmap** [comment]
In the released version 0.7.0, output directory is configurable if you needed.

```scala
Compile / jooqSource := crossTarget.value / "src_managed_jooq"
```
